Mathlib/Analysis/Calculus/Deriv/Add.lean

```diff
@@ -120,6 +120,18 @@ theorem deriv_const_add (c : F) : deriv (fun y => c + f y) x = deriv f x := by
 theorem deriv_const_add' (c : F) : (deriv fun y => c + f y) = deriv f :=
   funext fun _ => deriv_const_add c
 
+lemma differentiableAt_comp_const_add {a b : 𝕜} :
+    DifferentiableAt 𝕜 (fun x ↦ f (b + x)) a ↔ DifferentiableAt 𝕜 f (b + a) := by
+  refine ⟨fun H ↦ ?_, fun H ↦ H.comp _ (differentiable_id.const_add _).differentiableAt⟩
+  convert DifferentiableAt.comp (b + a) (by simpa)
+    (differentiable_id.const_add (-b)).differentiableAt
+  ext
+  simp
+
+lemma differentiableAt_comp_add_const {a b : 𝕜} :
+    DifferentiableAt 𝕜 (fun x ↦ f (x + b)) a ↔ DifferentiableAt 𝕜 f (a + b) := by
```

Comment:
Do we want iff versions like for the other ones below?

Reply:
I decided against them because they would involve subtraction, but I could be convinced otherwise

Reply:
Well, I'll leave it up to you.

Reply:
Added :)